### PR TITLE
Add fortify

### PR DESCRIPTION
**Pull request by apereverzin**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by mdayican — COMMENTED** [review]
where did you get this value?

**Comment by apereverzin** [comment]
From their example PR

**Review by mdayican — COMMENTED** [review]
is this ID specific to us?

**Comment by apereverzin** [comment]
Not sure

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -14,22 +14,22 @@ def channel = '#bsp-build-notices'
...
-    secret('idam-client-secret', 'IDAM_CLIENT_SECRET'),
-    secret('payments-staging-queue-send-shared-access-key', 'PAYMENTS_QUEUE_WRITE_ACCESS_KEY')
-  ]
+        'bulk-scan-${env}': [
+                secret('s2s-secret-payment-processor', 'S2S_SECRET'),
+                secret('idam-users-bulkscan-username', 'IDAM_USERS_BULKSCAN_USERNAME'),
+                secret('idam-users-bulkscan-password', 'IDAM_USERS_BULKSCAN_PASSWORD'),
+                secret('idam-client-secret', 'IDAM_CLIENT_SECRET'),
+                secret('payments-staging-queue-send-shared-access-key', 'PAYMENTS_QUEUE_WRITE_ACCESS_KEY')
+        ]
```
Comment (mdayican) [inline Jenkinsfile_CNP:32]: I think not correct formatting

**Comment by mdayican** [comment]
we do not need

**Pull request closed without merging**